dependencies/pip/dev_requirements.txt

```diff
@@ -188,8 +188,6 @@ django-redis==5.2.0
     # via -r dependencies/pip/requirements.in
 django-redis-sessions==0.6.2
     # via -r dependencies/pip/requirements.in
-django-registration-redux==2.10
```

Comment:
That's odd. I suppose this is a mistake from a prior merge. In the future, please write about this kind of thing in the PR description so that I know it was intentional.